doc/script_commands.txt

```diff
@@ -11359,3 +11359,11 @@ Note: Only works with classes that use the ranking system.
 Returns true if the player is dead else false.
 
 ---------------------------------------
+
+*is_autoloot({<char_id>});
```

Comment:
Maybe it's me, but `is_autoloot` seems a bit confusing since this almost sounds like you want to check if the item is an autoloot type or something. Maybe more like `is_autolooting` or `has_autoloot`?